libs/sql-parser/src/main/java/io/crate/sql/Identifiers.java

```diff
@@ -81,14 +92,22 @@ public static String quote(String identifier) {
      * i.e. when it contain a double-quote, has upper case letters or is a SQL keyword
      */
     public static String quoteIfNeeded(String identifier) {
-        if (quotesRequired(identifier)) {
+        return quoteIfNeeded(identifier, false);
+    }
+
+    public static String quoteIfNeeded(String identifier, boolean isFunction) {
+        if (quotesRequired(isFunction, identifier)) {
             return quote(identifier);
```

Comment:
Looks like we always pass true (ie this used only for functions). Maybe can drop the flag and refltec that in the name, `quiteFunctionIfNeeded`?

Reply:
Right, updated.